Zend/zend_builtin_functions.h

```diff
@@ -22,12 +22,12 @@
 
 #include "zend_types.h"
 
-typedef struct _zval_struct zval;
+struct _zval_struct;
```

Comment:
Stupid question maybe, but why do we need this at all? The typedef is defined in zend_types.h which is included right above?

Reply:
Ah, this one is indeed probably unneeded.